src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/store/oidc.ts

```diff
@@ -15,7 +15,7 @@ export const useOidcStore = defineStore("oidc", {
     userManager: new UserManager(useConfigStore().kcOidcConfiguration),
   }),
   actions: {
-    async oidcUserInfo(): Promise<Components.Schemas.UserInfo> {
+    async oidcUserInfo(): Promise<any> {
```

Comment:
just wanted to confirm that we want this to be any?

Reply:
I might change the user info object in ECER-1166 and roll back this to user info based on server schema, but for this story we need to show the user address and that's why i changed it to any